web_app/testSite.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <link rel="stylesheet" type="text/css" href="src/form.css">
+
+</head>
+<body>
+    <div id="task-modal" class="modal">
+        <div class="modal-content">
+        <span class="close">&times;</span>
+        <form id="frame">
+            <h1 id = "title">Create new Task</h1>
+            <div id="form-body">
+                <br>
+                <div class="params">
+                    <div class="desc">Task Name</div>   
+                    <div class="input-item">
```

Comment:
Can this work if you just have the `input` and not the `div`? (here and elsewhere)

Reply:
I've tried it but it didn't work for me before. That's the reason I stuck it in a div. If you have any ideas though, lmk.